core/storage/merkle/polkadot_trie_db/polkadot_node.cpp

```diff
@@ -21,10 +21,21 @@ namespace kagome::storage::merkle {
     uint16_t bitmap = 0u;
     for (auto i = 0u; i < kMaxChildren; i++) {
       // NOLINTNEXTLINE(cppcoreguidelines-pro-bounds-constant-array-index)
-      if (children[i] != nullptr) {
+      if (children[i]) {
         bitmap = bitmap | 1u << i;
       }
     }
     return bitmap;
   }
+
+  uint8_t BranchNode::childrenNum() const {
+    uint8_t count = 0;
```

Comment:
```
return std::count_if(children.begin(), children.end(), [](auto &&child) { return child; });
```

Reply:
```
return std::count_if(children.begin(), children.end(), [](auto &&child) { return child != nullptr; });
```

Reply:
condition should be explicit `child != nullptr`